packages/opentelemetry-api/package.json

```diff
@@ -10,18 +10,17 @@
   },
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "compile": "tsc --build",
-    "clean": "tsc --build --clean",
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "compile": "../../node_modules/.bin/tsc --build",
```

Comment:
Doing this at work as well and colleague pointed out the following perhaps you would like to do the same: `../../node_modules/.bin/<bin>` is noisy, this may be slightly nicer: 

```
  "basebin": "../../node_modules/.bin",
  "scripts": {
...
    "lint": "${npm_package_basebin}/eslint . --ext .ts",
    "clean": "${npm_package_basebin}/tsc --build --clean",
...
```

wdyt?